e2e_tests/tests/config.py

```diff
@@ -14,9 +14,9 @@
 
 
 DEFAULT_TF1_CPU_IMAGE = "determinedai/environments:py-3.7-pytorch-1.7-tf-1.15-cpu-6218891"
-DEFAULT_TF2_CPU_IMAGE = "determinedai/environments:py-3.8-pytorch-1.12-tf-2.8-cpu-6218891"
+DEFAULT_TF2_CPU_IMAGE = "determinedai/environments-dev:py-3.8-pytorch-1.12-tf-2.12-cpu-0.21.2"
 DEFAULT_TF1_GPU_IMAGE = "determinedai/environments:cuda-10.2-pytorch-1.7-tf-1.15-gpu-6218891"
-DEFAULT_TF2_GPU_IMAGE = "determinedai/environments:cuda-11.3-pytorch-1.12-tf-2.8-gpu-6218891"
```

Comment:
non-blocking: what does the `6218891` represent? Is it a naming convention we've moved away from?

Reply:
That's a commit hash in the `environments` repo. It's a finer granularity than using the Determined version.

Reply:
Thanks!